packages/react-text/src/stories/Text.stories.tsx

```diff
@@ -1,15 +1,18 @@
 import { makeStyles } from '@fluentui/react-make-styles';
 import * as React from 'react';
-import { Text } from './Text';
-import type { TextProps } from './Text';
+import { Meta } from '@storybook/react';
+import { Text } from '../Text';
+import type { TextProps } from '../Text';
+import textDescriptionMd from './TextDescription.md';
+import textBestPracticesMd from './TextBestPractices.md';
 
 const useStyles = makeStyles({
   container: {
     width: '100px',
   },
 });
 
-export const TextStory = (props: TextProps) => {
+export const Default = (props: TextProps) => {
```

Comment:
Please put each story in its own file, so it’d work well with export to codesandbox